docs/user/guides/canonical-urls.rst

```diff
@@ -0,0 +1,32 @@
+How to Enable Canonical URLs
+============================
+
+In this guide, we introduce relevant settings for enabling canonical URLs in popular documentation frameworks.
+
+.. seealso::
+
+   If you want to customize the domain from which your documentation project is served, please refer to :doc:`/guides/custom-domains`.
+
+Sphinx
+~~~~~~
+
+If you are using :doc:`Sphinx </intro/getting-started-with-sphinx>`,
+Read the Docs will automatically add a default value of the html_baseurl_ setting matching your canonical domain.
+
+If you have defined ``html_baseurl`` in your ``conf.py``, you need to ensure that the value is correct.
```

Comment:
What is `correct`? Should we recommend that they remove it and let us set it?

Reply:
I wondered about this, too. This is mostly what the old text says. Will update :+1:

Reply:
Now it's very explicit, it's actually nice because it hopefully motivates people to fix it.

Side-note that we opened a new issue today about adding an environment variable for the canonical URL, `#9781`